src-tauri/src/recorder/utils.rs

```diff
@@ -1,7 +1,7 @@
 use crate::AppState;
 use rand::Rng;
 use scap::{
-    capturer::{CGPoint, CGRect, CGSize, Capturer, Options, Resolution},
+    capturer::{Point, Area, Size, Capturer, Options, Resolution},
```

Comment:
accounts for this [PR from scap ](https://github.com/helmerapp/scap/pull/85/files) changes
 
 We can reject this in case of conflict . main should already have these.
 
 But I still face the same problem with scaling factor 🤔.